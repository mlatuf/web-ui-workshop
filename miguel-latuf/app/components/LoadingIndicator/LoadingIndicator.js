/* eslint-disable no-unused-vars */
import React from 'react';
import './style.scss';
import { LinearProgress } from '@material-ui/core';
import { makeStyles } from '@material-ui/core/styles';

const useStyles = makeStyles(theme => ({
    progress: {
        flexGrow: 1,
        marginTop: 10
    }
}));

const LoadingIndicator = () => {
    const classes = useStyles();
    return <LinearProgress className={classes.progress} />;
};

export default LoadingIndicator;
