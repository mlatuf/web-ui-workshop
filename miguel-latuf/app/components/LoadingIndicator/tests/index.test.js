import React from 'react';
import TestRenderer from 'react-test-renderer';
import LoadingIndicator from '../index';

describe('<LoadingIndicator />', () => {
  it('render the loader correctly', () => {
    const wrapper = TestRenderer.create(<LoadingIndicator />);
    expect(wrapper).toBeDefined();
    expect(wrapper).toMatchSnapshot();
  });
});
