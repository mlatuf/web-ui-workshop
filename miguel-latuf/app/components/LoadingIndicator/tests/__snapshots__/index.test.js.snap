// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LoadingIndicator /> render the loader correctly 1`] = `
<div
  className="MuiLinearProgress-root makeStyles-progress-1 MuiLinearProgress-colorPrimary MuiLinearProgress-indeterminate"
  role="progressbar"
>
  <div
    className="MuiLinearProgress-bar MuiLinearProgress-bar1Indeterminate MuiLinearProgress-barColorPrimary"
    style={Object {}}
  />
  <div
    className="MuiLinearProgress-bar MuiLinearProgress-bar2Indeterminate MuiLinearProgress-barColorPrimary"
    style={Object {}}
  />
</div>
`;
